includes/woopay-user/class-woopay-save-user.php

```diff
@@ -63,7 +63,7 @@ public function register_checkout_page_scripts() {
 			'WCPAY_WOOPAY',
 			'woopayCheckout',
 			[
-				'PRE_CHECK_SAVE_MY_INFO' => $account_data['pre_check_save_my_info']
+				'PRE_CHECK_SAVE_MY_INFO' => isset( $account_data['pre_check_save_my_info'] ) ? $account_data['pre_check_save_my_info'] : false,
```

Comment:
Nit: Here we could also use the null coalescing operator, but feel free to merge.

```php
$account_data['pre_check_save_my_info'] ?? false
```